src/components/Aside/Sidebar.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { AiOutlineDown, AiOutlineUp } from 'react-icons/ai';
+
+import { firestore } from '../../shared/api/firebase';
+import { Link, useParams } from 'react-router-dom';
+import { theme } from '../../styles/Theme';
+
+interface MenuSectionProps {
+  id: number;
+  title: string;
+  items: MenuItem[];
+  url?: string;
+}
+
+interface MenuItem {
+  text: string;
+  timeStamp: string;
+  url: string;
+}
+
+interface ItemContainerProps {
+  $isUrl: boolean;
+}
+
+const SidebarContainer = styled.div`
+  min-width: 256px;
+  height: 100vh;
+  background-color: ${theme.blueBg3};
+  color: ${theme.gray300};
+  padding: 24px 20px 0px;
+  position: fixed;
```

Comment:
저는 스타일 코드를 적을 때, 비슷한 놈들끼리 묶어서 적고 개행을 넣어주는 걸 좋아합니다. 

```suggestion
  position: fixed;

  min-width: 256px;
  height: 100vh;
      
  padding: 24px 20px 0px;
  
  background-color: ${theme.blueBg3};
  color: ${theme.gray300};
```

무슨 차이가 있냐면, 일단 스타일 코드만 읽어봐도 머릿속에 ui가 대충 그려지는 효과가 있습니다.
그리고 비슷한 걸 담당하는 애들끼리 모여있으니까 위에서부터 편하게 읽을 수 있다는 점도 좋습니다.
저는 개인적으로 적을 때

```css
.foo {
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;

  display: flex;
  flex-direction: column;
  justify-contents: center;
  align-items: center;

  width: 100px;
  height: 100px;

  padding: 100px;
  margin: 100px;

  background: red;

  border: 1px solid red;
  border-radius: 100px;
  
  font-family: 'Noto';
  font-size: 100px;
  line-height: 100px;
  font-weight: 400;
  color: red;  

  // etc
}
```

이런 순서대로 적습니다.
한 번 적용해보세요! 개행을 넣으면 코드가 길어지는 단점이 있는데, 그런 걸 방지하려고 저는 styles.ts 파일에 styled-component를 모아두고 import 해서 씁니다.